support-frontend/assets/pages/[countryGroupId]/validation.ts

```diff
@@ -18,4 +18,5 @@ export function preventDefaultValidityMessage(
  * This uses a Unicode character class escape
  * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape
  */
-export const doesNotContainEmojiPattern = '^[^\\p{Emoji_Presentation}]+$';
+export const doesNotContainEmojiOrWhitespacePattern =
+	'^(?!.*[\\p{Extended_Pictographic}\\s]).*$';
```

Comment:
Perhaps a comment here to explain the move from `Emoji_Presentation` to `Extended_Pictographic`? Looks like it would pickup an Emoji with a number in front of it which perhaps was a loophole previously.

Reply:
It's hard to write an authoritative comment on this I think, based on the googling I've done it's a bit confusing. But this definitely catches more